content/2_Bronze/Intro_Sets.mdx

```diff
@@ -673,10 +670,11 @@ unlikely to work, as it will probably cause a
 See the following snippet for an example:
 
 ```java
-void iterate_remove_set_BAD() {
+void iterateRemoveBad() {
 	Set<Integer> s = new TreeSet<Integer>();
```

Comment:
```suggestion
 	Set<Integer> s = new TreeSet<>();
```

Reply:
For some reason this change suggestion is bugged and github won't let me commit it - I'll just do the rest manually